WWDC/SessionsSplitViewController.swift

```diff
@@ -27,13 +27,17 @@ final class SessionsSplitViewController: NSSplitViewController {
         detailViewController = SessionDetailsViewController(listStyle: listStyle)
 
         super.init(nibName: nil, bundle: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(syncSplitView(notification:)), name: .sideBarSizeSyncNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(syncSplitView), name: .SideBarSizeSync, object: nil)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
From apple documentation
> If your app targets iOS 9.0 and later or macOS 10.11 and later, you don't need to unregister an observer in its dealloc method.

As we are targeting macOS 10.12, these global removals on deinit are superfluous.

Reply:
An ok, good to know. Those are weird ass semantics.